### ISSUE TITLE
MustardUI disappears from project after purging model and appending a new one.

### ISSUE THREAD

**Issue by mokujinh** [Mustard2/MustardUI#140]
As the title says, I updated a model and tried to importing it to a scene I was already working on with that same model. Before hand I cleaned the scene and deleted and purged "unused data". After appending the same model (newer version) the UI is nowhere to be found.

My steps to recreate the error were:

>Open blend file I saved the project.

>Keyframed a pose to load it into a new model.

>made the action a "fake user" so it won't be purged.

>Deleted (manually from the viewport) model and rig, then deleted all the collection leftovers along with their objects inside them.

>Manually purged the rig again Blender file display mode as I noticed the UI made the rig as fake user in earlier versions making it impossible to purge with the regular option. 

>Then went to File>Clean Up>Recursive Unused Data-blocks.

>Appended the newer model version

>clicked on the armature and loaded the action

>looked for the menu options, the menu is nowhere to be found.

**Comment by Mustard2** [comment]
Hi! This is what happens when you are using the model in viewport selection, and removing the model might create a state where the None armature is selected and therefore nothing is drawn in the UI.

If you enable Debug mode there is a new button in the addon settings. Can you try to click on that and see if the UI appears?

![image](https://github.com/Mustard2/MustardUI/assets/60780948/d962560a-baef-4dca-bae3-3734ad18a36f)

**Comment by mokujinh** [comment]
That did solve the problem, but only after a few tries for some reason, but it did solved it thanks.

**Comment by Mustard2** [comment]
I will close this then, if there are similar issues, open a new issue. :)

**Comment by Mustard2** [comment]
Need to check the clean function, maybe there's a step missing in restoring the correct viewport status.
